fastchat/serve/api_provider.py

```diff
@@ -147,3 +147,71 @@ def palm_api_stream_iter(chat, message, temperature, top_p, max_new_tokens):
             "error_code": 0,
         }
         yield data
+
+def ai2_api_stream_iter(model_name,
+                        messages,
+                        temperature,
+                        top_p,
+                        max_new_tokens,
+                        api_key=None,
+                        api_base=None,
+                        ):
+                        
+    from requests import post
+    from json import loads
+
+    # get keys and needed values
+    ai2_key = api_key or os.environ.get('AI2_API_KEY')
+    api_base = api_base or "https://inferd.allen.ai/api/v1/infer"
+    model_id = "mov_01hhndc7e8k2s9x379ge8sm8xs"
```

Comment:
```suggestion
    model_id = "mov_01hj1pj8aqn5s6f0zz1qb1chhn"
```

FYI @natolambert. This includes the fix for the `temperature = 0.0` issue.

Reply:
I have one more update coming. Stay tuned :). We made a bunch of other changes that we're releasing today.

Reply:
See https://github.com/natolambert/FastChat/pull/1